cmake_modules/code-coverage.cmake

```diff
@@ -0,0 +1,559 @@
+#
```

Comment:
Could you add a comment with an url where this is from? And maybe which version/git hash?

Reply:
added